setup/productionize/persistence/orca-sql.md

```diff
@@ -84,6 +84,27 @@ executionRepository:
 monitor:
   activeExecutions:
     redis: false
+
+# Use SQL for Orca's work queue
+# Settings from Netflix and may require adjustment for your environment
+# Only validated with AWS Aurora MySQL 5.7
+# Please PR if you have success with other databases
+keiko:
+  queue:
+    fillExecutorEachCycle: true
```

Comment:
Could be removed if this is to merge: https://github.com/spinnaker/keiko/pull/151

Reply:
Merged: https://github.com/spinnaker/keiko/releases/tag/v3.8.1

Reply:
Thanks, removed it from here then.